m4/josm.m4

```diff
@@ -0,0 +1,42 @@
+AC_DEFUN([JOSM_INIT],[
+  AC_SUBST(HAS_JOSM) 
+
+  AC_ARG_WITH(josm,
+    [AS_HELP_STRING([--with-josm],[Include support for validation/cleaning with JOSM (ARG=yes or no)])],
```

Comment:
This just checks for the JNI header, which gets installed with Java. Since the external josm jar gets pulled down by Maven, not checking for that here.